### PR TITLE
Update dev-guide-sample-application-python-sqlalchemy.md

### PR DESCRIPTION
**Pull request by RakerZh**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add SQLAlchemy docs link on the side can help users find more information and reference about implementation of the code when they look up query since this document file is about SQLAlchemy.
Searching multiple columns is one of the most cases users will use, so they don't have to look up official docs. However, if they want more complex cases as reference, they might usually follow the official docs, which is the link I attached.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): #15938 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Icemap** [comment]
@RakerZh 您好，可以讲一下这样更改的意图吗？

**Comment by RakerZh** [comment]
orm sqlalchemy 使用方式和 sql语句还是有不同的，建议用户看官方文档更好

**Comment by Icemap** [comment]
了解，但是这里呢？这和原示例不符，和其他示例不一致，也不符合输出。

> ```python
> query='player'
> more_players = session.query(Player).filter(Player.name.like("%"+query+"%"),Player.coins==10000).all()
> ```

**Comment by RakerZh** [comment]
这个修改是我觉得是当用户查询文档时，需要基本的增删改查方法，而多个column查询以及 正则表达式的方法我觉得是基本的使用方法之一，这样用户可以不需要再点开orm的官方文档进一步操作。我可以增加一些详细说明会好一些？我不知道官方文档页面关于这部分内容多少会比较好，毕竟有sqlalchemy的文档在那里

File: develop/dev-guide-sample-application-python-sqlalchemy.md [hunk 1]
```diff
@@ -239,17 +239,20 @@ with Session() as session:
...
+更多信息参考[插入数据](/develop/dev-guide-insert-data.md)，[SQLAlchemy Query](https://docs.sqlalchemy.org/en/20/orm/queryguide/index.html)。
 
 ### 查询数据
 
 ```python
 with Session() as session:
     player = session.query(Player).filter_by(name == "test").one()
+    query='player'
+    more_players = session.query(Player).filter(Player.name.like("%"+query+"%"),Player.coins==10000).all()
+
```
Comment (Icemap) [inline develop/dev-guide-sample-application-python-sqlalchemy.md:251]:
```suggestion
```

那这样是否可行呢？我们外链 SQLAlchemy 的官方文档后，删除这个特殊的示例。因为其他的文档中也不曾出现这个 `模糊匹配` 的示例。

Comment (RakerZh) [reply develop/dev-guide-sample-application-python-sqlalchemy.md:251]: 我觉得可以

Comment (qiancai) [reply develop/dev-guide-sample-application-python-sqlalchemy.md:251]: @RakerZh，如果可以的话，麻烦请提交下修改建议
image

**Comment by rpaik** [comment]
@RakerZh do you need any help with feedback 👆 ?

File: develop/dev-guide-sample-application-python-sqlalchemy.md [hunk 2]
```diff
@@ -239,7 +239,7 @@ with Session() as session:
     session.commit()
 ```
 
-更多信息参考[插入数据](/develop/dev-guide-insert-data.md)。
+更多信息参考[插入数据](/develop/dev-guide-insert-data.md)，[SQLAlchemy Query](https://docs.sqlalchemy.org/en/20/orm/queryguide/index.html)。
```
Comment (qiancai) [inline develop/dev-guide-sample-application-python-sqlalchemy.md:242]:
```suggestion
更多信息参考[插入数据](/develop/dev-guide-insert-data.md)以及 [SQLAlchemy Query](https://docs.sqlalchemy.org/en/20/orm/queryguide/index.html)。
```

File: develop/dev-guide-sample-application-python-sqlalchemy.md [hunk 3]
```diff
@@ -249,7 +249,7 @@ with Session() as session:
     print(player)
 ```
 
-更多信息参考[查询数据](/develop/dev-guide-get-data-from-single-table.md)。
+更多信息参考[查询数据](/develop/dev-guide-get-data-from-single-table.md), [SQLAlchemy Query](https://docs.sqlalchemy.org/en/20/orm/queryguide/index.html)。
```
Comment (qiancai) [inline develop/dev-guide-sample-application-python-sqlalchemy.md:252]:
```suggestion
更多信息参考[查询数据](/develop/dev-guide-get-data-from-single-table.md)以及 [SQLAlchemy Query](https://docs.sqlalchemy.org/en/20/orm/queryguide/index.html)。
```

File: develop/dev-guide-sample-application-python-sqlalchemy.md [hunk 4]
```diff
@@ -260,7 +260,7 @@ with Session() as session:
     session.commit()
 ```
 
-更多信息参考[更新数据](/develop/dev-guide-update-data.md)。
+更多信息参考[更新数据](/develop/dev-guide-update-data.md), [SQLAlchemy Query](https://docs.sqlalchemy.org/en/20/orm/queryguide/index.html)。
```
Comment (qiancai) [inline develop/dev-guide-sample-application-python-sqlalchemy.md:263]:
```suggestion
更多信息参考[更新数据](/develop/dev-guide-update-data.md)以及 [SQLAlchemy Query](https://docs.sqlalchemy.org/en/20/orm/queryguide/index.html)。
```

File: develop/dev-guide-sample-application-python-sqlalchemy.md [hunk 5]
```diff
@@ -271,7 +271,7 @@ with Session() as session:
     session.commit()
 ```
 
-更多信息参考[删除数据](/develop/dev-guide-delete-data.md)。
+更多信息参考[删除数据](/develop/dev-guide-delete-data.md), [SQLAlchemy Query](https://docs.sqlalchemy.org/en/20/orm/queryguide/index.html)。
```
Comment (qiancai) [inline develop/dev-guide-sample-application-python-sqlalchemy.md:274]:
```suggestion
更多信息参考[删除数据](/develop/dev-guide-delete-data.md)以及 [SQLAlchemy Query](https://docs.sqlalchemy.org/en/20/orm/queryguide/index.html)。
```

**Review by qiancai — APPROVED** [review]

**Review by Icemap — APPROVED** [review]
LGTM

**Pull request merged**